### PR TITLE
WIP: add nextclade annotations to metadata

### PR DESCRIPTION
**Pull request by rneher**
First attempt to include nextclade annotation in metadata.

this doesn't yet run, as it needs merging of the CLI PRs that support zip datasets and positional arguments for the sequences to be processed. It also needs a not yet released MPXV data set.

### CODE REVIEW

**Comment by rneher** [comment]
this should now work with CLI 2.0-beta and the now released data sets!

**Comment by jameshadfield** [comment]
This ran smoothly for me using nextclade 2.0.0-alpha.13. Few minor observations:

* Short sequences are skipped by nextclade (e.g. "Unable to align: no seed matches.") with the side effect that their respective metadata fields are blank. We might want to add `QC_missing_data=bad` or similar via [join-metadata-and-clades.py](https://github.com/nextstrain/monkeypox/pull/58/files#diff-323b767605a7a3b09b262932d563a6e202608f12c3c1a898e00af13443a1f12d)?
* Leaves `mpxv.zip` as an untracked file

**Comment by jameshadfield** [comment]
I also had to change the nextalign invocation after upgrading to alpha 13:
```diff
--- a/workflow/snakemake_rules/core.smk
+++ b/workflow/snakemake_rules/core.smk
@@ -81,12 +81,12 @@ rule align:
         """
         nextalign run \
             --jobs {threads} \
-            --sequences {input.sequences} \
-            --reference {input.reference} \
+            --input-ref {input.reference} \
             --max-indel {params.max_indel} \
             --seed-spacing {params.seed_spacing} \
             --output-fasta {output.alignment} \
-            --output-insertions {output.insertions}
+            --output-insertions {output.insertions} \
+            {input.sequences}
         """
```

**Review by joverlee521 — APPROVED** [review]
I left minor comments but LGTM.

File: ingest/workflow/snakemake_rules/nextclade.smk [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+rule nextclade_dataset:
+    output:
+        "mpxv.zip"
```
[the remark this answers is not in the document]
Comment (corneliusroemer) [reply ingest/workflow/snakemake_rules/nextclade.smk:4]: done

**Comment by corneliusroemer** [comment]
Is it on purpose that frameshifts contain all the frameshifts with detailed annotation rather than a count? I think a count would be less verbose and easier to use in filter.

Also we now have two clades with the same name so one gets called `clade_x` the other `clade_y` by `pd.merge`.

Should we drop the old geographical clades?

**Pull request merged**